domain/policy_config.go

```diff
@@ -0,0 +1,34 @@
+package domain
+
+type ConfigMatchApplication struct {
+	Kind      string `json:"kind"`
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+}
+
+type ConfigMatchResource struct {
+	Kind      string `json:"kind"`
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+}
+
+type PolicyConfigTarget struct {
```

Comment:
I suggest we make this PolicyConfigMatch so it's consistent with the application and the resource and also to be more intuitive in the `PolicyConfig` struct especially with the json being `match`